pom.xml

```diff
@@ -116,7 +116,7 @@
     <htrace.version>3.1.0-incubating</htrace.version>
     <collections.version>3.2.2</collections.version>
     <!-- Do not change jodatime.version until HBASE-15199 is fixed -->
-    <jodatime.version>1.6</jodatime.version>
+    <jodatime.version>2.0</jodatime.version>
```

Comment:
Took a look at the HBASE-15199, seems like a classpath issue for jruby mostly.  Probably demo safe but i saw a phoenix issue with TO_DATE as well in the links.